### DISCUSSION TITLE
Is it posible to exclude ABI while packaging aab?

### DISCUSSION THREAD

**Discussion by bobrokrol** [airsdk/Adobe-Runtime-Support#2081] [Q&A]
I have a ANR / crashes on google testing devices  for x86 / x64 platforms after uploading to internal testing.
I dont want to support x86_64  but aab tool automaticaly ad alll possible ABIs
Is it possible to  left only arm64-v8a, armeabi-v7a ?
Any workarounds? I have numerous af ane's that I can't recompile

**Comment by ajwfrost** [comment]
If you use the "android-studio" target to generate an Android Studio project rather than directly generating the .aab file, you can then open it up in Android Studio and edit the 'app' folder's `build.gradle` file. Then within the `android` section you want something like:
```
    splits {
        abi {
            enable false
            reset()
            include "armeabi-v7a","arm64-v8a"
        }
    }
```
Then you can export a release bundle or do whatever else is required...

I'm not 100% sure on this though - the "splits" thing is normally for splitting into separate APK files - so there may be another Gradle setting for this. I'm 99% sure it's possible though. (Or perhaps, ignore that section, and just delete the files from the lib folder under the x86 and x86_86 folders!)

thanks

**Reply by bobrokrol** [reply]
> 

Thank you it works! Not the eassiest way of course. It would be great to have this option in adt

**Reply by ajwfrost** [reply]
Yes, there's actually a new build setting coming in our next SDK release that will make it a bit more trivial to do this...

FYI I found that the "splits" thing is only for APK development, for AAB it should be something like:
```
android {
    ...
    defaultConfig {
        applicationId "... "
        ...
        ndk {
            abiFilters "arm64-v8a", "armeabi-v7a"
        }
    }
}
```
(or just take away the unnecessary folders of course)

thanks

**Reply by bobrokrol** [reply]
Thank you, I've just removed related ABI  folders in the android project
